src/components/typography/font-roboto-examples.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+
+import FontRobotoExamples from './font-roboto-examples';
+
+export default {
+  title: 'Components/Typography',
+  component: FontRobotoExamples,
+};
+
+const Template = (args) => <FontRobotoExamples {...args} />;

```

Comment:
Not sure if having a story for every different font is the best approach, rather we should have have a story for typography were `font-family` is a dropdown menu option?

Reply:
Unless we do have a configurable one & the specific ones and I just haven't gotten there yet.   It might be best to organize with a `stories` folder so that we can see all the type stories in once space, and the code is not mixed in with code that is meant to be library (exportable, consumable) code.

Reply:
Following up on this. It is probably best to organize the folder like this:

`docs`: folder container the `TypePage`, `TypographyTop`, `TypeCard`, `TextSample`, etc. - they are all building blocks, in a sense, for the "Types" page in the toolkit which exists as a documentation really, not necessarily as something we want exported in our toolkit.
    - Within `docs` may even want to `components` folder to put all the components and then leave the stories under `docs` (or the other way around).   

Part of the goal here is that we can ignore all `docs` folders when bundling the library.

Then under `Typography` I suppose you'd just have `index.ts`, and  I think you should add a `Typography.stories.ts` which is just using the `Typography` component with different configurations the user can select from the Controls (e.g., font, size)

Reply:
Hi Troy, yes it's better to put all what I did into the docs folder, they are only for the instuctions actually.

Reply:
- The two font examples stories have benn combined into one story.
- All the files relating to the typography have been moved to docs/components folder.